docs/security.md

```diff
@@ -37,26 +40,35 @@ Both stores must be base64 encoded, for example:
+
+```bash
+dcos security secrets create /truststore --value-file trust.jks.base64
+dcos security secrets create /keystore --value-file server.jks.base64
+```
 
-    {
-      "security": {
-        "ssl": {
-          "enabled": true,
-          "keyStoreBase64": "/u3+7QAAAAIAAAACAAAAAgA...”,
-          "keyStorePassword": "secret",
-          "keyPassword": "secret"
-        }
-      }
-    }
+In this case you're adding two secrets `/truststore` and `/keystore` that you will need to pass to the Spark Driver and Executors. You will need to add the following configurations to your `dcos spark run ` command:
 
-Install Spark with your custom configuration:
+```bash
 
-    dcos package install --options=options.json spark
+dcos spark run --verbose --submit-args="\
+--conf spark.mesos.containerizer=mesos \  # use mesos containerizer
+--conf spark.ssl.enabled=true \
+--conf spark.ssl.enabledAlgorithms=TLS_RSA_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_256_CBC_SHA \
+--conf spark.ssl.keyPassword=<key password> \
+--conf spark.ssl.keyStore=server.jks \  # This MUST be set this way
+--conf spark.ssl.keyStorePassword=<keystore access password> \
+--conf spark.ssl.protocol=TLS \
+--conf spark.ssl.trustStore=trust.jks \  # this MUST be set this way
+--conf spark.ssl.trustStorePassword=<truststore password> \
+--conf spark.mesos.driver.labels=DCOS_SECRETS_DIRECTIVE:[{\"name\"\:\"/keystore\"\,\"type\"\:\"ENVIRONMENT\"\,\"environment\"\:{\"name\"\:\"KEYSTORE_BASE64\"}}\,{\"name\"\:\"/truststore\"\,\"type\"\:\"ENVIRONMENT\"\,\"environment\"\:{\"name\"\:\"TRUSTSTORE_BASE64\"}}] \
+--conf spark.mesos.task.labels=DCOS_SECRETS_DIRECTIVE:[{\"name\"\:\"/keystore\"\,\"type\"\:\"ENVIRONMENT\"\,\"environment\"\:{\"name\"\:\"KEYSTORE_BASE64\"}}\,{\"name\"\:\"/truststore\"\,\"type\"\:\"ENVIRONMENT\"\,\"environment\"\:{\"name\"\:\"TRUSTSTORE_BASE64\"}}],DCOS_SPACE:/spark, \
+--class <Spark Main class> <Spark Application JAR> [application args]"
+```
 
-Make sure to connect the DC/OS cluster only using an SSL connection (i.e. by using an `https://<dcos-url>`). Use the following command to set your DC/OS URL:
+Importantly the `spark.mesos.driver.labels` and `spark.mesos.task.labels` must be set as shown. If you upload your secret with another path (e.g. not `/keystore` and `/truststore`) then change the `name` in the value accordingly. Lastly, `spark.mesos.task.lables` must have the `DCOS_SPACE:<dcos_space>` label as well, to have access to the secret. See the [Secrets Documentation about SPACES][13] for more details about Spaces, but usually you want `/spark` as shown.
 
-    dcos config set core.dcos_url https://<dcos-url>
 
  [11]: https://docs.mesosphere.com/1.9/overview/architecture/components/
  [12]: http://docs.oracle.com/javase/8/docs/technotes/tools/unix/keytool.html
+ [13]: https://docs.mesosphere.com/service-docs/spark/v2.0.1-2.2.0-1/run-job/
```

Comment:
As referenced above, this link is supposed to point to "Secrets Documentation about SPACES".